projects/Directory.Packages.props

```diff
@@ -46,4 +47,4 @@
     <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
     <GlobalPackageReference Include="MinVer" Version="5.0.0" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Happy to revert if you prefer keeping this.

Reply:
I have no idea how these files end up without the final newline 🤷

Reply:
It might be Visual Studio code cleanup.

Reply:
Rider does it automatically for some reason